core/src/solvers/pipeline_planner.cpp

```diff
@@ -67,11 +67,17 @@ PipelinePlanner::PipelinePlanner() {
 	                    planning_pipeline::PlanningPipeline::MOTION_PLAN_REQUEST_TOPIC);
 }
 
+PipelinePlanner::PipelinePlanner(const planning_pipeline::PlanningPipelinePtr& planning_pipeline) : PipelinePlanner() {
+	planner_ = planning_pipeline;
+}
+
 void PipelinePlanner::init(const core::RobotModelConstPtr& robot_model) {
-	planner_ = Task::createPlanner(robot_model);
+	if (!planner_) {
+		planner_ = Task::createPlanner(robot_model);
 
-	planner_->displayComputedMotionPlans(properties().get<bool>("display_motion_plans"));
-	planner_->publishReceivedRequests(properties().get<bool>("publish_planning_requests"));
+		planner_->displayComputedMotionPlans(properties().get<bool>("display_motion_plans"));
```

Comment:
We should probably still apply those settings somehow

Reply:
Thanks! I thought they are parameters that we should load to the parameter server